lib/Controller/MainController.php

```diff
@@ -119,7 +119,7 @@ public function search($query)
 			$recipes = $this->service->findRecipesInSearchIndex($query);
 
 			foreach ($recipes as $i => $recipe) {
-                $recipes[$i]['image_url'] = $this->urlGenerator->linkToRoute(
+                $recipes[$i]['imageUrl'] = $this->urlGenerator->linkToRoute(
```

Comment:
I am a bit surprised about this change. Could you have mixed up with another bug fixure?

Reply:
I have actually explained this before to someone, but cannot find my comment anywhere (which is odd). In any case, I changed the front-end to use imageUrl instead of image_url at some point, so this change was necessary. I still can't remember why I did it, but I have a feeling it was to fix some bug. Or maybe I just wanted to move the JSON API to use more JSON-like attribute names, but didn't finish the job. I admit there is some discrepancy in the attribute naming and it should be resolved by either (a) reverting the imageUrl back to PHP style image_url (a step in the wrong direction, in my opinion) or (b) changing the rest of the JSON API attribute names to camelCase (would require extensive work and testing). In any case, I feel this is something to address in an issue of its own.

Reply:
You are rigght. I just wanted to make sure this is not used anywhere else.